### DISCUSSION TITLE
How can I avoid that $owner_password is set?

### DISCUSSION THREAD

**Discussion by blskop** [mpdf/mpdf#1780] [General]
In your [documentation](https://mpdf.github.io/reference/mpdf-functions/setprotection.html) there is only one possibility for $owner_password and that is that a password is always set as I understand.

Either I set a password myself or if I omit it/set it to zero then mpdf generates a password itself.

This is a big problem for us because we use your Libary to create invoices and automated invoicing systems from our customers think that there is a password problem.

In the following screenshot you can see that the "Security Method" is set to Password Security and "Permissions Password" is set to Yes.

![nuoSE](https://user-images.githubusercontent.com/95758592/208629598-f05bc45e-a61b-4b87-bb02-3a370606f6dd.png)

The "Security Method" must be set to No Security and this only happens if this "Permissions Password" is not set.

We use the following code to set the protection:

$mpdf->SetProtection(array('print','copy','modify','annot-forms','fill-forms','extract','assemble','print-highres'));

I hope you can help us with our problem and happy holidays :)

**Comment by ssieb** [comment]
Those permissions are useless anyway.  Why are you trying to set them?

**Reply by blskop** [reply]
I don't want to set them...I would like them not to be set but that is not possible in my eyes at the moment.

**Reply by ssieb** [reply]
Then why are you calling `SetProtection` at all?
